Tests/CommandsTests/BuildToolTests.swift

```diff
@@ -506,7 +506,7 @@ final class BuildToolTests: CommandsTestCase {
                 return buildArenaPath.appending(component: filename)
             }
 
-            let dummySwiftcPath = SwiftPM.testBinaryPath(for: "dummy-swiftc")
+            let dummySwiftcPath = SwiftPM.xctestBinaryPath(for: "dummy-swiftc")
```

Comment:
The only uses of this code are around tests of XCTest integration, yes. I renamed the function to reflect that it only makes sense for XCTest. Because swift-testing is an experimental feature and is not part of the toolchain, our ability to write tests of swift-testing in SwiftPM is limited.